src/apps/material/material.tsx

```diff
@@ -1,43 +1,45 @@
-import React, { useEffect, useState } from "react";
-import VariousIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Various.svg";
 import CreateIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Create.svg";
 import Receipt from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Receipt.svg";
+import VariousIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/Various.svg";
+import React, { useEffect, useState } from "react";
 import { useDeepCompareEffect } from "react-use";
-import { WorkId } from "../../core/utils/types/ids";
+import DisclosureControllable from "../../components/Disclosures/DisclosureControllable";
+import DisclosureSummary from "../../components/Disclosures/DisclosureSummary";
+import DigitalModal from "../../components/material/digital-modal/DigitalModal";
+import InfomediaModal from "../../components/material/infomedia/InfomediaModal";
+import { hasCorrectAccess } from "../../components/material/material-buttons/helper";
```

Comment:
The changes to the dependencies are not relevant to the change at hand. I expect it to be the result of a local plugin or the like which orders dependencies by internal/external ad then alphabetically. This is not really something that is part of our general setup/code style.

I do not mind the change as such but next time around I would like to see it in a separate commit.

If it is something that we would like to enforce in general we should move it to our eslint/prettier setup.

Reply:
That's fair. For the record, i think something uniform should be added and enforced then, but i won't do it as part of this 👍